### PR TITLE
CLDC-2619: migrate staging data changes

### PR DESCRIPTION
**Pull request by Chirag-Bhatti**
- Fixes required for successfully migrating the staging DB
- Setting create db migration infra to true for staging

### CODE REVIEW

File: db-migration/run.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ fi
 
 echo "Setting up conduit into Gov PaaS DB and making dump file"
 
-if cf conduit ${CF_SERVICE} -- pg_dump -v -j 2 --file dumpfile --no-acl --encoding utf8 --clean --no-owner --if-exists -Fc; then
+if cf conduit ${CF_SERVICE} -- pg_dump -v --file dumpfile --no-acl --encoding utf8 --clean --no-owner --if-exists -Fc; then
```
Comment (Chirag-Bhatti) [inline db-migration/run.sh:14]: Can't use workers with the -Fc option (which is an option to use the custom file format for pg_dump)

File: db-migration/run.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 echo 'Logging into Gov PaaS'
 
-if cf login -a api.london.cloud.service.gov.uk -u ${CF_USERNAME} -p ${CF_PASSWORD} -s ${CF_SPACE}; then
+if cf login -a api.london.cloud.service.gov.uk -o "dluhc-core" -u ${CF_USERNAME} -p ${CF_PASSWORD} -s ${CF_SPACE}; then
```
Comment (Chirag-Bhatti) [inline db-migration/run.sh:5]: Don't necessarily need the org here (now that we realised it was that the logon didn't have access to the staging space causing the issue, not this) but there's no harm either

File: terraform/modules/ecr_migration/main.tf [hunk 3]
```diff
@@ -9,6 +9,7 @@ terraform {
   }
 }
 
+#tfsec:ignore:aws-ecr-enforce-immutable-repository:mutable images preferred
```
Comment (RachaelBooth) [inline terraform/modules/ecr_migration/main.tf:12]: This looks like a duplicate of two lines below?

Comment (Chirag-Bhatti) [reply terraform/modules/ecr_migration/main.tf]: Whoops good spot! I think I missed this when I merged the latest changes in from main (about s3-migration). Removed now 😀

**Review by RachaelBooth — APPROVED** [review]

**Pull request merged**